views/js/apipCreator/helper/jquery.xpath.min.js

```diff
@@ -0,0 +1,13 @@
ho){0y(!ho7x18x)3y 2x;7y{8y jY=ho[0];0y(jY 1y ct)jY=cW7x38x(jY);8z(8y fz=1,fG=ho7x18x,jX;fz<fG;fz2v){jX=ho[fz];0y(jX 1y ct)jX=cW7x38x(jX);jY=ea['+'](jY,jX,5x)}3y eh['div'](jY,2y c8(fG),5x)}3z(e){4y e7x678x9w_[9]?e:2y X(_[70])}});eL(_[194],[[bK,'*'],[cm,'',3x8x,9z(ho,go){0y(!ho7x18x)3y 2x;7y{8y jY=ho[0];8z(8y fz=1,fG=ho7x18x;fz<fG;fz2v)0y(ee['ge'](ho[fz],jY,5x)7x28x2w)jY=ho[fz];3y jY}3z(e){4y e7x678x9w_[9]?e:2y X(_[70])}});eL(_[186],[[bK,'*'],[cm,'',3x8x,9z(ho,go){0y(!ho7x18x)3y 2x;7y{8y jY=ho[0];8z(8y fz=1,fG=ho7x18x;fz<fG;fz2v)0y(ee['le'](ho[fz],jY,5x)7x28x2w)jY=ho[fz];3y jY}3z(e){4y e7x678x9w_[9]?e:2y X(_[70])}});eL(_[510],[[bK,'*'],[bK,'?',3x8x,9z(ho,hI){0y(!ho7x18x){0y(4x7x18x>1)3y hI;7z 3y 2y cW(0);3y 2x}7y{8y jY=ho[0];0y(jY 1y ct)jY=cW7x38x(jY);8z(8y fz=1,fG=ho7x18x,jX;fz<fG;fz2v){jX=ho[fz];0y(jX 1y ct)jX=cW7x38x(jX);jY=ea['+'](jY,jX,5x)}3y jY}3z(e){4y e7x678x9w_[9]?e:2y X(_[70])}});eL(_[515],[[cm,'*'],[cA,'',3x8x,9z(ho,gX){0y(4x7x18x<2){0y(!5x7x248x7x358x(5x7x268x))4y 2y X(_[9]);gX=5x7x268x}8y gw=ej7x1608x7x168x(5x,gX);0y(5x7x248x7x288x(gw,_[27])9w9)4y 2y X(_[195]);8y hn=0w;8z(8y fz=0;fz<ho7x18x;fz2v)8z(8y fZ=0,r=eP(ho[fz])7x518x(/\\s+/),f0=r7x18x;fZ<f0;fZ2v)0y((gX=5x7x248x7x1178x(gw,r[fZ]))3wcF(hn,gX)6w-1)hn7x118x(gX);3y d5(hn,5x)});eL(_[197],[[cm,'*'],[cA,'',3x8x,9z(ho,gX){4y \"Function '\"+_[197]+\"' not implemented\"});eL(_[198],[[cm,'?',3x8x,9z(hC){4y \"Function '\"+_[198]+\"' not implemented\"});eL(_[199],[[cm,'?',3x8x,9z(hC){4y \"Function '\"+_[199]+\"' not implemented\"});eL(_[193],[[cm,'?',3x8x,9z(hC){4y \"Function '\"+_[193]+\"' not implemented\"});eL(_[192],[[cm,'*'],[cA,'',3x8x,9z(ho,gX){4y \"Function '\"+_[192]+\"' not implemented\"});9z d6(ho,gr){0y(!ho7x18x)3y 1x;8y gK=ho[0];0y(gr7x248x7x358x(gK))3y 3x;0y(ho7x18x6w1){0y(gK 1y bP)3y gK7x78x7x28x2w;0y(gK 1y cm)3y7wgK7x28x2w7x18x;0y(eW(gK))3y!(dd(gK7x28x2w)4wgK7x28x2w6w0);4y 2y X(_[70])}4y 2y X(_[70])};9z d4(ho,gr){8y hn=0w;8z(8y fz=0,fG=ho7x18x,gK,jU;fz<fG;fz2v){gK=ho[fz];jU=2x;0y(gK6w2x)jU=2x;7z 0y(gr7x248x7x358x(gK)){8y d8=gr7x248x7x288x;6y(d8(gK,_[27])){2z 1:jU=2y ct(d8(gK,_[93]));1z;2z 2:jU=2y ct(d8(gK,_[7]));1z;2z 3:2z 4:2z 8:jU=2y ct(d8(gK,_[121]));1z;2z 7:jU=2y ct(d8(gK,_[121]));1z;2z 9:8y gX=d8(gK,_[331]);jU=2y ct(gX?d8(gX,_[93]):'');1z}}7z 0y(gK 1y bK)jU=gK;0y(jU9w2x)hn7x118x(jU)}3y hn};9z d5(ho,gr){3y ho7x4528x(9z(gX,gZ){8y fS=gr7x248x7x658x(gX,gZ);3y fS&2?1:fS&4?-1:0})};eL(_[516],[[c8,'*'8x,9z(ho){8y r=0w;8z(8y fz=0,fG=ho7x18x;fz<fG;fz2v)r7x118x(bz7x4418x(ho[fz]));3y 2y cm(r7x548x(''))});eL(_[522],[[cm,'?'8x,9z(hD){0y(hD6w2x)3y 2x;8y jK=hD7x28x2w;0y(jK6w'')3y 0w;8y hn=0w;8z(8y fz=0,fG=jK7x18x;fz<fG;fz2v)hn7x118x(2y c8(jK7x2298x(fz)));3y hn});eL(_[48],[[cm,'?'],[cm,'?'],[cm,'',3x8x,9z(hE,hF,go){0y(hE6w2x4whF6w2x)3y 2x;8y ih=5x7x508x7x1838x,jS;0y(4x7x18x>2)ih=go7x28x2w;jS=ih6wit+_[188]?gn:5x7x508x7x2358x(ih);0y(!jS)4y 2y X(_[191]);3y 2y c8(jS7x488x(hE7x28x2w,hF7x28x2w))});eL(_[523],[[cm,'?'],[cm,'?'8x,9z(hE,hF){0y(hE6w2x4whF6w2x)3y 2x;3y 2y bP(hE7x28x2w6whF7x28x2w)});
```

Comment:
I am a bit confused, when trying to merge this branch, I realize that this file is not being used anywhere.